app/models/user.rb

```diff
@@ -1207,10 +1207,22 @@ def secondary_emails
     self.user_emails.secondary.pluck(:email)
   end
 
+  RECENT_TIME_READ_THRESHOLD = 60.days
```

Comment:
```suggestion
  RECENT_TIME_READ_THRESHOLD ||= 60.days
```
This helps in case the class is reloaded for any reason.